cmd/humioctl/aggregate_alerts_export.go

```diff
@@ -65,3 +65,42 @@ func newAggregateAlertsExportCmd() *cobra.Command {
 
 	return &cmd
 }
+
+func newAggregateAlertsExportAllCmd() *cobra.Command {
+	var outputDirectory string
+
+	cmd := cobra.Command{
+		Use:   "export-all <view>",
+		Short: "Export all aggregate alerts",
+		Long:  `Export all aggregate alerts to yaml files with naming <sanitized-aggregate-alert-name>.yaml.`,
```

Comment:
Consider explaining what a sanitized name is. The same goes for all 4 new commands.